olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/eclipse-che-preview-kubernetes.package.yaml

```diff
@@ -3,5 +3,5 @@ channels:
 - name: stable
   currentCSV: eclipse-che-preview-kubernetes.v7.0.0-rc-2.0
 - name: nightly
-  currentCSV: eclipse-che-preview-kubernetes.v9.9.9-nightly.1563804656
+  currentCSV: eclipse-che-preview-kubernetes.v9.9.9-nightly.1563883405
```

Comment:
new file names include timestamp `1563883406` and `1563883405` ... should they all be the same? Makes it hard to review this PR when most of the content is new (but has different versioning).

Reply:
yes, timestamp is used to create semver-compatible nighty version numbers.
But you're right, we could use the same timestamp for both `kubernetes` and `openshift`
Maybe in another PR.